src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/Settings.java

```diff
@@ -34,7 +34,8 @@
                     "Task poll interval", SettingDefaults.TASK_POLL_INTERVAL, this::setTaskPollInterval, this::getTaskPollInterval),
             new Setting(PREFIX+".connection.endpoint", "The Cosmos DB endpoint.", "CosmosDB Endpoint", this::setEndpoint, this::getEndpoint),
             new Setting(PREFIX+".master.key", "The connection master key.", "Master Key", this::setKey, this::getKey),
-            new Setting(PREFIX+".cosmosdb.databasename", "The Cosmos DB target database.", "CosmosDB Database Name", this::setDatabaseName, this::getDatabaseName),
+            new Setting(PREFIX+".cosmosdb.databasename", "(Deprecated) The Cosmos DB target database.", "CosmosDB Database Name", this::setDatabaseName, this::getDatabaseName),

```

Comment:
i would just remove this (Deprecated). we will note this as a breaking change and will bump the version number when we merge and push a new release.